protocols/gossipsub/src/peer_score/params.rs

```diff
@@ -165,6 +170,9 @@ impl Default for PeerScoreParams {
             decay_interval: Duration::from_secs(DEFAULT_DECAY_INTERVAL),
             decay_to_zero: DEFAULT_DECAY_TO_ZERO,
             retain_score: Duration::from_secs(3600),
+            slow_peer_weight: -0.2,
+            slow_peer_threshold: 0.0,
+            slow_peer_decay: 0.2,
```

Comment:
cc @AgeManning

Reply:
They are made up as suggestions. 

These are rough estimates from what we expect on real networks. I.e They are minor penalties relative to the behaviour penalty. 

We didn't want to penalize peers too much, so having 50 failures to 1 behaviour penalty seemed reasonable.

Note that these are not on by default and it is generally recommended that users set their own scoring based on their own networks.

Reply:
> We didn't want to penalize peers too much, so having 50 failures to 1 behaviour penalty seemed reasonable.

OK, makes sense to me! Since this is defining the default for `PeerScoreParams`, it should be a safe and documented value (even if not enabled by default). IMO a comment saying that the `slow_peer_weight` is 50x lower than `behaviour_penalty_weight` would help.

Reply:
updated, thanks Gui!